README.md

```diff
@@ -1,57 +1,215 @@
-# Project Name
+# Extracting, transforming, and loading data with ACI
 
-(short, 1-3 sentenced, description of the project)
+In this sample, we're trying to do some basic analysis on the [HappyDB](https://github.com/rit-public/HappyDB) dataset, by doing word count and drawing word cloud, to highlight the words appear more frequently in the people's happy moments.
```

Comment:
Suggested change with grammatical fixes:
"In this sample, we're analyzing the HappyDB dataset by creating a word cloud of the most frequent words in people's self-reported happiest moments. We'll extract a data source, load it into a database, and transform the data into something we can render graphically."

Reply:
fixed